index.css

```diff
@@ -78,8 +88,372 @@
     margin-left: 10%;
   }
 
+  .works-centered img {
+    width: 100%;
+  }
+
+  .attach-icon img {
+    float: left;
+    width: 5%;
+    height: 5%;
+    margin-right: 2%;
+  }
+
   .social a img {
     width: 19.99px;
     height: 19.96px;
   }
+
+  .extra {
+    margin-left: 6%;
+    margin-right: 6%;
```

Comment:
- [ ] Kindly use pixels instead percentages for the margins to be similar to the Figma design.